pkg/webhook/server/certificate/manager.go

```diff
@@ -117,7 +117,7 @@ func (m *Manager) Start(stopCh <-chan struct{}) error {
 // of generating new one
```

Comment:
Hey what will happen if cert-manager is restarted and current CA is dummy cert
will it trigger new cert creation?
@RamLavi @qinqon